spec/keys_spec.rb

```diff
@@ -130,6 +130,13 @@ module FakeRedis
       expect(@client.exists("key1")).to be false
     end
 
+    it "should get integer and string keys" do
+      @client.set("key1", "1")
```

Comment:
I could also write a spec where it just handles `integer` keys `[1, 2]` returning `["1", "2"]`